releasenotes/notes/update-cs-csdg-in-equivalence-library-c1e70d3246f4aa6d.yaml

```diff
@@ -0,0 +1,5 @@
+---
+fixes:
+  - |
+    Update the synthesis of :class:`.CSGate` and :class:`.CSdgGate` in the equivalence library
+    so that the basis translator will use only two :class:`.CXGate`s for their synthesis and not 3.
```

Comment:
`CSGate` was already using 2 only right?

Reply:
yes, but now it uses less 1-qubit gates, I'll fix this